### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by OR13**

### CODE REVIEW

File: draft-ietf-jose-fully-specified-algorithms.xml [hunk 1]
```diff
@@ -534,26 +534,50 @@
 	</section>
       </section>
 
-      <section anchor="UpdatedInstructions" title="Updated Review Instructions for Designated Experts">
+<section anchor="UpdatedInstructions" title="Updated Review Instructions for Designated Experts">
+  <section anchor="UpdatedInstructions1" title="JSON Web Signature and Encryption Algorithms">
+	<t>
+		IANA is directed to preserve the current Reference to RFC7518,
+		and to add a reference to this section, of this document.
```
Comment (selfissued) [inline draft-ietf-jose-fully-specified-algorithms.xml]:
```suggestion
		IANA is directed to preserve the current reference to RFC 7518,
		and to add a reference to this section of this document.
```

File: draft-ietf-jose-fully-specified-algorithms.xml [hunk 2]
```diff
@@ -534,26 +534,50 @@
...
 	  <list style="symbols">
 	    <t>
-	      Registration requests for polymorphic algorithm identifiers must not be accepted;
-	      only fully-specified algorithm identifiers may be registered going forward.
+				Only fully-specified algorithm identifiers may be registered.
+	      Polymorphic algorithm identifiers must not be registered.
+	    </t>
+			<t>
+				Only proposed standard RFCs are acceptable as references.
 	    </t>
```
Comment (selfissued) [inline draft-ietf-jose-fully-specified-algorithms.xml]:
```suggestion
```

File: draft-ietf-jose-fully-specified-algorithms.xml [hunk 3]
```diff
@@ -534,26 +534,50 @@
...
-	  are also updated to add the same additional review criterion.
+	  have been updated to include an additional review criterion:
+	  <list style="symbols">
+	    <t>
+				Only fully-specified algorithm identifiers may be registered.
+	      Polymorphic algorithm identifiers must not be registered.
+	    </t>
+			<t>
+				Only proposed standard RFCs are acceptable as references.
+	    </t>
```
Comment (selfissued) [inline draft-ietf-jose-fully-specified-algorithms.xml]:
```suggestion
```

File: draft-ietf-jose-fully-specified-algorithms.xml [hunk 4]
```diff
@@ -534,26 +534,50 @@
...
+			<t>
+				Only proposed standard RFCs are acceptable as references.
 	    </t>
 	  </list>
 	</t>
+	</section>
+	<section anchor="UpdatedInstructions2" title="COSE Algorithms">
+	<t>
+		IANA is directed to preserve the current Reference to RFC9053 and RFC9054,
+		and to add a reference to this section, of this document.
```
Comment (selfissued) [inline draft-ietf-jose-fully-specified-algorithms.xml]:
```suggestion
		IANA is directed to preserve the current references to RFC 9053 and RFC 9054,
		and to add a reference to this section of this document.
```

**Review by selfissued — CHANGES_REQUESTED** [review]
Please apply my suggestions.

**Comment by OR13** [comment]
I have applied your suggestions.

**Review by selfissued — APPROVED** [review]

**Pull request merged**